docs/demos/TrafficManagerProfile/session-build-multi-clsuter-app/README.md

```diff
@@ -0,0 +1,221 @@
+# Deploying a Multi-Cluster Application using fleet APIs
+
+## Introduction 🚀
+In this session, we will walk through how to deploy a Nginx server seamlessly across multiple production clusters in **eastus2euap** while routing traffic intelligently using **Azure Traffic Manager**.
+
+![](build-multi-cluster-app.png)
+
+## Prerequisites
+Before we begin, ensure you have the following:
+- An **Azure Kubernetes Fleet Manager** set up with two members.
+- The **kubectl** CLI installed and configured.
+- A resource group created for the Traffic Manager Profile.
+- Azure Traffic Manager permission set up for **Azure Kubernetes Fleet Manager**.

```

Comment:
Hi Zhiying! Just a general Q, not a blocker: do we need to give people directions/instructions for this step?

Reply:
Will create a separate doc for this (probably an Azure doc)